### DISCUSSION TITLE
Multiplayer 'ironman' - Create new upon death

### DISCUSSION THREAD

**Discussion by ProphetCrow** [FakeFishGames/Barotrauma#10961] [Ideas and suggestions]
A server setting that forces deceased crew members to "create new" upon death. 

And maybe another adjustable setting to adjust talent EXP gain rate to offset the penalty of death

**Comment by Ek-Videogames** [comment]
This is the default multiplayer behavior though?
I'm guessing you mean have it as a server setting that can be toggled on and off, with off being the "old behavior" that was briefly around when talents & reaper's tax was implemented. I agree in that regard, leaving it as as setting for server hosts to decide is good.
